app/src/main/kotlin/co/netguru/baby/monitor/client/feature/communication/pairing/ServiceDiscoveryFragment.kt

```diff
@@ -20,14 +21,13 @@ import co.netguru.baby.monitor.client.feature.communication.nsd.NsdServicesAdapt
 import co.netguru.baby.monitor.client.feature.communication.nsd.NsdState
 import co.netguru.baby.monitor.client.feature.communication.nsd.ResolveFailedException
 import co.netguru.baby.monitor.client.feature.communication.nsd.StartDiscoveryFailedException
-import co.netguru.baby.monitor.client.feature.settings.ConfigurationViewModel
 import io.reactivex.Single
 import io.reactivex.disposables.Disposable
 import kotlinx.android.synthetic.main.fragment_connecting_devices.*
 import java.util.concurrent.TimeUnit
 import javax.inject.Inject
 
-class ConnectingDevicesFragment : BaseDaggerFragment() {
+class ServiceDiscoveryFragment : BaseDaggerFragment() {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/9259abfa8c2adcd3ca5c84779794ca9bdaf7e360/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/communication/pairing/ServiceDiscoveryFragment.kt#L30"></span>Class ‘ServiceDiscoveryFragment’ with ‘15’ functions detected. Defined threshold inside classes is set to ‘15’</td>
    </tr>
  </tbody>
</table>